### PR TITLE
Add time_to_live argument for instances

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24361
related pr: https://github.com/demisto/content-test-conf/pull/865

## Description
Added a script which takes time to live argument from user and set the instances to live according to this time (this script can be used on any branch).
In addition this script can get contributor branch argument which takes the changes from the remote branch and run content build with the contributor's changes.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -61,6 +61,26 @@ jobs:
...
+              echo "Skipping, contributor branch not given."
+              exit 0
+            else
+              USER=$(echo $CONTRIB_BRANCH | cut -d ":" -f 1)
+              BRANCH=$(echo $CONTRIB_BRANCH | cut -d ":" -f 2)
+
+              echo 'Copy the changes from the contributor branch'
+              git remote add $USER git@github.com:$USER/content.git
+              git fetch $USER $BRANCH
+              git checkout -t $USER/$BRANCH
```
Comment (ronykoz) [inline .circleci/config.yml:80]: better to take his changes i think, for instance take a list of packs

Comment (barchen1) [reply .circleci/config.yml]: done

File: .circleci/config.yml [hunk 2]
```diff
@@ -350,14 +373,18 @@ jobs:
                   then
                     export IFRA_ENV_TYPE="Demisto PreGA" # disable-secrets-detection
 
+                elif [ -n "${CONTRIB_BRANCH}" ] ;
+                  then
+                    export IFRA_ENV_TYPE="Content-GA" # disable-secrets-detection
```
Comment (yaakovi) [inline .circleci/config.yml:378]: better to use marketplace instance

File: .circleci/config.yml [hunk 3]
```diff
@@ -143,6 +143,27 @@ jobs:
           fingerprints:
             - "02:df:a5:6a:53:9a:f5:5d:bd:a6:fc:b2:db:9b:c9:47" # disable-secrets-detection
             - "f5:25:6a:e5:ac:4b:84:fb:60:54:14:82:f1:e9:6c:f9" # disable-secrets-detection
+      - run:
+          name: Check if Contributor branch
```
Comment (ronykoz) [inline .circleci/config.yml:147]:
```suggestion
          name: Get Contributor pack
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -143,6 +143,27 @@ jobs:
...
+          when: always
+          command: |
+            echo $CONTRIB_BRANCH
```
Comment (ronykoz) [inline .circleci/config.yml:150]: remove

File: .circleci/config.yml [hunk 5]
```diff
@@ -143,6 +143,27 @@ jobs:
...
+            if [ -z $CONTRIB_BRANCH ]
+            then
+              echo "Skipping, contributor branch not given."
+              exit 0
+            else
+              USER=$(echo $CONTRIB_BRANCH | cut -d ":" -f 1)
+              BRANCH=$(echo $CONTRIB_BRANCH | cut -d ":" -f 2)
+              PACK=$(echo $CONTRIB_BRANCH | cut -d ":" -f 3)
+
+              echo 'Copy the changes from the contributor branch'
```
Comment (ronykoz) [inline .circleci/config.yml:160]:
```suggestion
              echo 'Copy the changes from the contributor branch $USER/$BRANCH in the pack $PACK'
```

File: .circleci/config.yml (hunk 3 above)
Comment (barchen1) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 4 above)
Comment (barchen1) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 5 above)
Comment (barchen1) [reply .circleci/config.yml]: done

**Pull request merged**